sdk/eventhub/azure-eventhub/azure/eventhub/_client_base.py

```diff
@@ -307,19 +268,27 @@ def get_partition_properties(self, partition_id):
             )
         return output
 
-    def close(self):
+    def _close(self):
         # type:() -> None
         self._conn_manager.close_connection()
 
 
 class ConsumerProducerMixin(object):
+    def __init__(self):
```

Comment:
A mixin must not have a `__init__`, since it will break 
If you have mypy issue, you can do like this:
https://github.com/Azure/azure-sdk-for-python/blob/ebd73c3fc22dcf17be2a903f32bdd95d9090f283/sdk/core/azure-core/azure/core/tracing/_abstract_span.py#L188-L194

And make the base class the on that contains _name only during type-checking. If that's not possible because the one that contains _name cannot be easily initialize, then that's a fair type ignore.

Reply:
Changed to use this way. Thanks.